pkg/server/proto/server.proto

```diff
@@ -1258,6 +1264,15 @@ message ListTaskResponse {
   repeated GetTaskResponse tasks = 1;
 }
 
+message CancelTaskRequest {
+  // A reference to the task to cancel
+  Ref.Task ref = 1;
+
+  // Forcefully attempt to cancel the task
+  // TODO(briancain): Unclear if this is a good idea
```

Comment:
I would add more commentary on this, or remove it completely to avoid confusing folks.

Reply:
Agreed. I think given the caution we have around forcefully canceling jobs, I don't like the idea of making it easier to cancel 3 jobs via the Task API. If someone really wants to they can do a manual force cancel the hard way. I'll remove this!